addons/calendar_bis/models/calendar_event.py

```diff
@@ -107,6 +110,23 @@ def merge_timeslots(self, values, ts_to_skip=None):
         (to_unlink + current).unlink()
         return to_create + values
 
+    def exdate(self, timeslot):
+        self.ensure_one()
+        self.exdated = self.exdated + ',' + str(timeslot.start.date()) if self.exdated else str(timeslot.start.date())
```

Comment:
I guess we don't need conditions here

Reply:
default is '', so if we exdate ad date, we end up with `,<date>` and we might as well have nice strings

Reply:
Ah, I was getting the wrong order. I assumed it was `self.exdated + ',' + (x if self.exdated else x)` :facepalm: